google_guest_agent/hostname/hostname_windows.go

```diff
@@ -0,0 +1,120 @@
+//  Copyright 2019 Google Inc. All Rights Reserved.
```

Comment:
Nit: `2024` instead of `2019.

Reply:
Fixed license headers and build directives.